services/121-service/src/programs/programs.service.ts

```diff
@@ -226,9 +227,6 @@ export class ProgramService {
       await queryRunner.release();
     }
 
-    // Loop through FSPs again to store config, which can only be done after program is saved
```

Comment:
this comment was outdated. You cannot store the fsp config when creating a program